README.md

```diff
@@ -1,264 +1,135 @@
+|`ENABLE_DESTROY` | Bool | `true` in dev, `false` elsewhere |<p>If `true`, deploys a pipeline which, when run, will completely destroy Cloud Foundry and all of it's data</p><p>This should absolutely **never** be set to `true` in a staging or production deployment</p>|
+
+## Accessing Concourse
+Once deployed, Concourse can be accessed from the URLs below. By default, authentication with Github is enabled.
+
+| Environment type | Environment name | URL |
+| ---------------- | ---------------- | --- |
+| Dev | Unique name | https://deployer.$NAME.dev.cloudpipeline.digital/ |
+| Dev | Dev[0-9]+ | https://deployer.dev$NUMBER.dev.cloudpipeline.digital/ |
+| Staging | `stg-lon` | https://deployer.london.staging.cloudpipeline.digital/ |
+| CI | `build` | https://concourse.build.ci.cloudpipeline.digital/ |
+| Production | `prod` | https://deployer.cloud.service.gov.uk/ |
+| Production | `prod-lon` | https://deployer.london.cloud.service.gov.uk/ |
+
+Non-development URLs are also accessible via the `gds paas open` command.
 
-## Pingdom checks
-Visit [Pingdom documentation page](https://github.com/alphagov/paas-cf/blob/master/doc/pingdom.md)
+## Finding configuration
+The following table outlines some important directories in the repository, their purpose, and when you might need to look in them.
 
-## Other useful commands
-Type `make` to get the list of all available commands.
+| Directory | Purpose | I will need this when .. |
+| -- | -- | -- |
+| `concourse/pipelines/` | YAML definitions of the Concourse pipelines | I want to make a change to how the platform is deployed, monitored, or torn down |
+| `config/billing/` | The scripts and static files used to generate configuration for the billing system.| <ul><li>I'm adding a new backing service, so that I can set how much it costs.</li><li>The VAT rate has changed </li><li>The cost of an AWS resource has changed</li></ul>|
+| `manifests/cf-manifest/` | The Bosh manifest configuration for Cloud Foundry | See specific directories below |
+| `manifests/cf-manifest/operations.d/` | Customisations applied to `cf-deployment`, applicable to all environments | <ul><li>I want to make a configuration change that will affect every environment</li><li>I want to deploy a new piece of software with a Bosh release</li></ul>
+| `manifests/cf-manifest/operations` | Customisations applied to `cf-deployment` [based on some condition](https://github.com/alphagov/paas-cf/blob/main/manifests/cf-manifest/scripts/generate-manifest.sh#L18) | I want to make a configuration change that will only be applied in certain circumstances |
+|`manifests/cf-manifest/spec`| Unit tests applied to the generated manifest file | I want to ensure a property of the manifest is not invalidated (for example, correct number of instances of some VM) |
```

Comment:
Should VM be plural here?

Reply:
Hmmm. Good question. I was trying to talk about ensuring we have the right number of instances of a component, but without getting in to all the nuance of being able to have more than 1 component on a machine. Happy to take guidance.

Reply:
Quick poll with the team suggests they prefer "some VM" over "some VMs" or anything else